recipes/pyviz/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "pyviz" %}
+{% set version = "0.10.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/{{name}}/{{name}}-{{ version }}.tar.gz
+  sha256: 1e0463c31219dcb5018d0bcf01cfc47eed46e7c9970ca1accc52dbb93da234d8
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - pyviz = pyviz.__main__:main
+  skip: True  # [py2k]
```

Comment:
Would it make sense to just make this `noarch` (especially given Python 3.6 is already required in `host` and will soon be required in `build`)?